component/prometheus/relabel/relabel.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/grafana/agent/component"
 	flow_relabel "github.com/grafana/agent/component/common/relabel"
 	"github.com/grafana/agent/component/prometheus"
+	lru "github.com/hashicorp/golang-lru"
```

Comment:
Did you meant to use `github.com/hashicorp/golang-lru/v2`, which supports generics?

Reply:
I didn't explicitly do that one since Loki isn't, but def can switch to it.

Reply:
Seems like it might be nicer for us, but I'll leave the choice up to you